index.html

```diff
@@ -604,6 +601,30 @@
   }).then(onSessionStarted);
 });
 
+const matrixUpdateListeners = {};
+function addMatrixUpdateListener(p) {
+  removeMatrixUpdateListener(p);
```

Comment:
When can this remove ever happen/be required?

Reply:
It protects against adding multiple matrixupdate listeners to the same package. I don't know the code well enough to know if that scenario could/will ever happen, but I don't think it hurts to have this check. Happy to remove it if necessary.

Reply:
As far as I know a package cannot be added twice without first being removed, but this check implies otherwise. Yeah, I would remove it.

Reply:
Removed